tensorflow/tools/ci_build/linux/rocm/run_gpu_single.sh

```diff
@@ -51,7 +51,22 @@ export TF_PYTHON_VERSION=$PYTHON_VERSION
 export TF_NEED_ROCM=1
 export ROCM_PATH=$ROCM_INSTALL_DIR
 
-yes "" | $PYTHON_BIN_PATH configure.py
```

Comment:
the configure should end up in the else path

Reply:
I don't understand what you mean. Did I nuke the wrong commit when managing the conflict?

Reply:
might be just a merge conflict/artifact. Either way the else ("legacy") path needs to call configure.

Reply:
I just added what is in that commit you wanted. The `configure.py` logic was already under the else path so still not sure what you mean - but I didn't have the rest of the else logic from that commit (deleted wrong stuff) so added it.